plugins/techdocs/src/home/components/EntityListDocsTable.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { useCopyToClipboard } from 'react-use';
+import { capitalize } from 'lodash';
+import {
+  CodeSnippet,
+  TableColumn,
+  TableProps,
+  WarningPanel,
+} from '@backstage/core-components';
+import {
+  favoriteEntityIcon,
+  favoriteEntityTooltip,
+  useEntityListProvider,
+  useStarredEntities,
+} from '@backstage/plugin-catalog-react';
+import { DocsTable } from './DocsTable';
+import { DocsTableRow } from './types';
+
+export const EntityListDocsTable = ({
+  columns,
+  actions,
+}: {
+  columns?: TableColumn<DocsTableRow>[];
+  actions?: TableProps<DocsTableRow>['actions'];
+}) => {
+  const { loading, error, entities, filters } = useEntityListProvider();
+  const { isStarredEntity, toggleStarredEntity } = useStarredEntities();
+  const [, copyToClipboard] = useCopyToClipboard();
+
+  const title = capitalize(filters.user?.value ?? 'all');
+
+  const defaultActions = [
+    DocsTable.actions.createCopyDocsUrlAction(copyToClipboard),
+    ({ entity }: DocsTableRow) => {
+      const isStarred = isStarredEntity(entity);
+      return {
+        cellStyle: { paddingLeft: '1em' },
+        icon: () => favoriteEntityIcon(isStarred),
+        tooltip: favoriteEntityTooltip(isStarred),
+        onClick: () => toggleStarredEntity(entity),
+      };
+    },
+  ];
+
+  if (error) {
+    return (
+      <WarningPanel
+        severity="error"
+        title="Could not load available documentation."
+      >
+        <CodeSnippet language="text" text={error.toString()} />
+      </WarningPanel>
+    );
+  }
+
+  return (
+    <DocsTable
+      title={title}
+      entities={entities}
+      loading={loading}
+      actions={actions || defaultActions}
+      columns={columns}
+    />
+  );
+};
+
+EntityListDocsTable.columns = DocsTable.columns;
```

Comment:
I believe it's this reassignment that's causing the API Reports issue, try switching to a `import * as actionFactories from './actions'` etc. instead

Reply:
ah that fixed it, thanks! out of curiosity, how did you debug this?

Reply:
Modified the `.d.ts` files in `dist-types` and ran `yarn build:api-reports:only` to see where the errors was triggered from and then applied some workarounds. More specifically I renamed the `columns.d.ts` file to break references to it, and then fixed them one by one to see what imports triggered the issue.

Reported it back to the project btw, so hopefully there'll be a fix so we can avoid future workarounds: https://github.com/microsoft/rushstack/issues/2842